metrics/config_observability_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2018 The Knative Authors.
```

Comment:
<!--copyright-bot-->
Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors.
```

Reply:
this file was moved from `knative.dev/serving/pkg/metrics/config_test.go`